pkg/constants/constants.go

```diff
@@ -70,34 +70,51 @@ const (
 
 	Disable = "disable"
 
-	TimeParseLayout    = "2006-01-02 15:04:05"
-	MinStakeDuration   = 24 * 14 * time.Hour
-	MaxStakeDuration   = 24 * 365 * time.Hour
-	MaxStakeWeight     = 100
-	MinStakeWeight     = 1
-	DefaultStakeWeight = 20
+	TimeParseLayout             = "2006-01-02 15:04:05"
+	MinMainnetStakeDuration     = 24 * 14 * time.Hour
+	MaxStakeDuration            = 24 * 365 * time.Hour
+	MinStakeWeight              = 1
+	DefaultStakeWeight          = 20
+	AVAXSymbol                  = "AVAX"
+	DefaultFujiStakeDuration    = "48h"
+	DefaultMainnetStakeDuration = "336h"
 	// The absolute minimum is 25 seconds, but set to 1 minute to allow for
 	// time to go through the command
-	StakingStartLeadTime               = 1 * time.Minute
-	StakingMinimumLeadTime             = 25 * time.Second
-	TerraformNodeConfigFile            = "node_config.tf"
-	AvalancheCLISuffix                 = "-avalanche-cli"
-	AWSDefaultCredential               = "default"
-	CertSuffix                         = "-kp.pem"
-	AWSSecurityGroupSuffix             = "-sg"
-	SSHTCPPort                         = 22
-	AvalanchegoAPIPort                 = 9650
-	AvalanchegoP2PPort                 = 9651
-	OutboundPort                       = 0
-	Terraform                          = "terraform"
-	AnsiblePlaybook                    = "ansible-playbook"
-	SetupNodePlaybook                  = "playbook/setupNode.yml"
-	AnsibleInventoryDir                = "inventories"
-	AnsibleInventoryFlag               = "-i"
-	AnsibleExtraArgsIdentitiesOnlyFlag = "--ssh-extra-args='-o IdentitiesOnly=yes'"
-	DefaultConfigFileName              = ".avalanche-cli"
-	DefaultConfigFileType              = "json"
-	WriteReadReadPerms                 = 0o644
+	StakingStartLeadTime                  = 1 * time.Minute
```

Comment:
I would like @dasconnor to consider on leading time for validators and posible issues from that (if any)

Reply:
Yeah, let's do 5 minutes for all